lighthouse-cli/chrome-launcher.ts

```diff
@@ -30,6 +30,12 @@ const log = require('../lighthouse-core/lib/log');
 const spawn = childProcess.spawn;
 const execSync = childProcess.execSync;
 
+const green = '\x1B[32m';
+const reset = '\x1B[0m';
+
+// See https://github.com/GoogleChrome/lighthouse/issues/1228
+const tick = process.platform === 'win32' ? '\u221A' : '✓';
```

Comment:
okay. then we'll stick with this hilarity. 

Let's pull `process.platform === 'win32'` out into a `isWindows` boolean then
 @XhmikosR , sg?